README.md

```diff
@@ -24,6 +24,18 @@ for Windows and Linux, in order to compile Lantern for OSX you'll need an OSX
 host, this is a limitation caused by Lantern depending on C code and OSX build
 tools for certain features.
 
+### Docker Installation Instructions
+
+1. Get the [Docker Toolbox](https://www.docker.com/docker-toolbox)
+2. Install docker per [these instructions](https://docs.docker.com/mac/step_one/)
+
+After installation, you'll have a docker machine called `default`, which is what the build script uses. You'll probably want to increase the memory and cpu for the default machine, which will require you to recreate it:
+
+```bash
+docker-machine rm default
+docker-machine create --driver virtualbox --virtualbox-cpu-count 2 --virtualbox-memory 4096 default
+```
+
```

Comment:
I'd suggest to add following lines because the command in https://docs.docker.com/machine/migrate-to-machine/ creates a machine named `docker-vm` instead.

To migrate from `boot2docker`:
```bash
docker-machine create -d virtualbox --virtualbox-import-boot2docker-vm boot2docker-vm default
```